clients/cli/src/orchestrator_client.rs

```diff
@@ -155,10 +160,14 @@ impl OrchestratorClient {
         task_id: &str,
         proof_hash: &str,
         proof: Vec<u8>,
+        signing_key: SigningKey,
     ) -> Result<(), Box<dyn std::error::Error>> {
         let (program_memory, total_memory) = get_memory_info();
         let flops = measure_gflops();
 
+        let verifying_key: VerifyingKey = signing_key.verifying_key();
+        let signature = signing_key.sign(proof_hash.as_bytes());
```

Comment:
Another todo here is to actually think about the structure of the object being signed -- what if any identifiers should be tied in, how do we guarantee domain separation once we start incorporating additional info, and so forth. But I think the minimalism of this is a virtue for the moment.

Reply:
Should the signature be on task_id || proof hash? That would prevent a bad actor from replaying the same signature over and over to knock another node off the network (they'd have to get the signature first, but signatures shouldn't be secrets)

Reply:
@sjudson Can you take another look? I've updated the signed message to include the signature version and task_id